src/Forms/ConfirmedPasswordField.php

```diff
@@ -72,79 +73,54 @@ class ConfirmedPasswordField extends FormField
      *
      * Caution: The form field does not include any JavaScript or CSS when used outside of the CMS context,
      * since the required frontend dependencies are included through CMS bundling.
-     *
-     * @param boolean $showOnClick
      */
-    protected $showOnClick = false;
+    protected bool $showOnClick = false;
 
     /**
      * Check if the existing password should be entered first
-     *
-     * @var bool
      */
-    protected $requireExistingPassword = false;
+    protected bool $requireExistingPassword = false;
 
 
     /**
      * A place to temporarily store the confirm password value
-     *
-     * @var string
      */
-    protected $confirmValue;
+    protected string $confirmValue = '';
 
     /**
      * Store value of "Current Password" field
-     *
-     * @var string
      */
-    protected $currentPasswordValue;
+    protected string $currentPasswordValue = '';
 
     /**
      * Title for the link that triggers the visibility of password fields.
-     *
-     * @var string
      */
-    public $showOnClickTitle;
+    public string $showOnClickTitle = '';
 
     /**
      * Child fields (_Password, _ConfirmPassword)
-     *
-     * @var FieldList
      */
-    public $children;
+    public FieldList $children;
 
     protected $schemaDataType = FormField::SCHEMA_DATA_TYPE_STRUCTURAL;
 
-    /**
-     * @var PasswordField
-     */
-    protected $passwordField = null;
+    protected ?PasswordField $passwordField;
 
-    /**
-     * @var PasswordField
-     */
-    protected $confirmPasswordfield = null;
+    protected ?PasswordField $confirmPasswordfield;
 
-    /**
-     * @var HiddenField
-     */
-    protected $hiddenField = null;
+    protected ?HiddenField $hiddenField = null;
 
     /**
-     * @param string $name
-     * @param string $title
-     * @param mixed $value
      * @param Form $form Ignored for ConfirmedPasswordField.
```

Comment:
We should probably remove this "ignored" property but I'm not doing it in this PR.